rules/rules.libsonnet

```diff
@@ -393,19 +393,269 @@
             ||| % $._config,
           },
           {
-              record: 'node:node_inodes_free:',
-              expr: |||
+            record: 'node:node_inodes_free:',
+            expr: |||
+              max(
                 max(
-                  max(
-                    kube_pod_info{%(kubeStateMetricsSelector)s, host_ip!=""}
-                  ) by (node, host_ip)
-                  * on (host_ip) group_right (node)
-                  label_replace(
-                    (max(node_filesystem_files_free{%(nodeExporterSelector)s, %(hostMountpointSelector)s}) by (instance)), "host_ip", "$1", "instance", "(.*):.*"
-                  )
-                ) by (node)
-              ||| % $._config,
-          },          
+                  kube_pod_info{%(kubeStateMetricsSelector)s, host_ip!=""}
+                ) by (node, host_ip)
+                * on (host_ip) group_right (node)
+                label_replace(
+                  (max(node_filesystem_files_free{%(nodeExporterSelector)s, %(hostMountpointSelector)s}) by (instance)), "host_ip", "$1", "instance", "(.*):.*"
+                )
+              ) by (node)
+            ||| % $._config,
+          },
+        ],
+      },
+      {
+        name: 'windows.node.rules',
```

Comment:
we should extract this into a separate file so it's optionally included like the suggestion I made about dashboards

Reply:
I have moved the windows specific rules to another file.